pkg/client/cli/license.go

```diff
@@ -0,0 +1,40 @@
+package cli
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"github.com/telepresenceio/telepresence/v2/pkg/client/cli/cliutil"
+)
+
+type licenseInfo struct {
+	outputFile string
+}
+
+func LicenseCommand() *cobra.Command {
+	li := &licenseInfo{}
+	cmd := &cobra.Command{
+		Use:  "license",
+		Args: cobra.MinimumNArgs(1),
+
+		Short: "Get License from Ambassador Cloud",
+		Long:  "Get License from Ambasssador Cloud",
```

Comment:
typo: `Ambasssador` has too many `s`

Reply:
Dumb question: How do I know what to pass to Telepresence as the license key ID?  I assume you're writing docs to explain that... but the `Long` help should at least point the user in the right direction to find that info.

Reply:
Yup, I'm just waiting for the UI page to be set in staging so I A. know it works B. can point users to how to use it.

Right now I get my license via the API but I'm not expecting users to do that... and there will be a more in-depth tutorial in the docs.